pkg/util/errors/errors.go

```diff
@@ -5,4 +5,5 @@ var (
 	ErrInitServicesNotReady    = New(1001, "failed to initialize. Services are not ready")
 	ErrTimeoutServicesNotReady = New(1002, "failed with timeout error.  Services are not ready")
 	ErrHealthCheck             = New(1003, "Periodic health checker failed.  Services are not ready")
+	ErrStatusUpdate            = New(1004, "periodic status updater failed.  Services are not ready")
```

Comment:
Can you combine 1003 and 1004 to one error and variablize the check name?

Reply:
also @kflan-io I didn't see any error mark downs in these commits